diff --git a/node_modules/ng-packagr/lib/styles/stylesheet-processor.js b/node_modules/ng-packagr/lib/styles/stylesheet-processor.js
index 48aa06c..73df9db 100644
--- a/node_modules/ng-packagr/lib/styles/stylesheet-processor.js
+++ b/node_modules/ng-packagr/lib/styles/stylesheet-processor.js
@@ -110,6 +110,7 @@ class StylesheetProcessor {
             // replicate the `initializePlugin` behavior from https://github.com/cssnano/cssnano/blob/a566cc5/packages/cssnano/src/index.js#L8
             .map(([creator, pluginConfig]) => creator(pluginConfig))
             .filter((plugin) => !asyncPlugins.includes(plugin.postcssPlugin));
+        postCssPlugins.push(require(process.cwd() + '/' + 'scripts/themeable/add-origin-varvalue.js' ));
         postCssPlugins.push(...cssNanoPlugins);
         return postcss(postCssPlugins);
     }
