diff --git a/node_modules/ng-packagr/lib/styles/stylesheet-processor.js b/node_modules/ng-packagr/lib/styles/stylesheet-processor.js
index aafb43a..cb97e57 100644
--- a/node_modules/ng-packagr/lib/styles/stylesheet-processor.js
+++ b/node_modules/ng-packagr/lib/styles/stylesheet-processor.js
@@ -105,6 +105,7 @@ class StylesheetProcessor {
                 },
             },
         }));
+        postCssPlugins.push(require(process.cwd() + '/' + 'scripts/themeable/add-origin-varvalue.js' ));
         return postcss(postCssPlugins);
     }
 }
