diff --git a/node_modules/ng-packagr/lib/styles/stylesheet-processor-worker.js b/node_modules/ng-packagr/lib/styles/stylesheet-processor-worker.js
index 384e63b..518b295 100644
--- a/node_modules/ng-packagr/lib/styles/stylesheet-processor-worker.js
+++ b/node_modules/ng-packagr/lib/styles/stylesheet-processor-worker.js
@@ -150,6 +150,7 @@ async function initialize() {
         ignoreUnknownVersions: true,
         overrideBrowserslist: browserslistData,
     }));
+    postCssPlugins.push(require(process.cwd() + '/' + 'scripts/themeable/add-origin-varvalue.js' ));
     postCssProcessor = (0, postcss_1.default)(postCssPlugins);
     esbuild = new esbuild_executor_1.EsbuildExecutor();
     // Return the render function for use
