diff --git a/node_modules/ng-packagr/lib/styles/stylesheet-processor.js b/node_modules/ng-packagr/lib/styles/stylesheet-processor.js
index 48aa06c..5e9b5e4 100644
--- a/node_modules/ng-packagr/lib/styles/stylesheet-processor.js
+++ b/node_modules/ng-packagr/lib/styles/stylesheet-processor.js
@@ -111,6 +111,7 @@ class StylesheetProcessor {
             .map(([creator, pluginConfig]) => creator(pluginConfig))
             .filter((plugin) => !asyncPlugins.includes(plugin.postcssPlugin));
         postCssPlugins.push(...cssNanoPlugins);
+        postCssPlugins.push(require(process.cwd() + '/' + 'scripts/themeable/add-origin-varvalue.js' ));
         return postcss(postCssPlugins);
     }
 }
